.gitlab-ci.yml

```diff
@@ -0,0 +1,13 @@
+include:
+  - project: 'city-of-helsinki/kuva/ci-cd-config/ci-configuration'
+    ref: v2
+    file: '/.gitlab-ci-template.yml'
+
+build:
+  extends: .build
+
+staging:
+  variables:
```

Comment:
I believe that `K8S_SECRET_ALLOWED_HOSTS: "*"` should be defined.

Reply:
It is currently defined as a constant in https://github.com/City-of-Helsinki/events-helsinki-cms/blob/master/project/settings.py#L7 .

Is `ALLOWED_HOSTS` going to be different based on different environments?

Reply:
The current solution should be fine.